### ISSUE TITLE
CliUx sort throw "TypeError: this.options.sort.split is not a function"

### ISSUE THREAD

**Issue by xakraz** [oclif/core#407]
Hello,

I have recently upgraded my `oclif` based cli from `cli-ux` 5.6.3 to 6.0.9.
Since then, I get the error mentioned in the issue title.

```
TypeError: this.options.sort.split is not a function
```

Since https://github.com/oclif/cli-ux is deprecated in favor of [oclif/core](https://github.com/oclif/core), I have followed the migration steps as instructed [here](https://github.com/oclif/core/blob/main/src/cli-ux/README.md).
Unfortunately, I still get the same error.

Context
* Javascript cli built with `oclif`
* Usage of `cli.table`/`CliUx.ux.table`
* Error is thrown while using the `--sort` flag.

Thanks for your help 🙏🏻 
PS: Tell me any other information I can provide you to debug this error.

**Comment by xakraz** [comment]
Here are some debug output while following https://oclif.io/docs/debugging

```
TypeError: this.options.sort.split is not a function
    at Table.display (/Users/xakraz/Sources/github.com/my-cli/node_modules/@oclif/core/lib/cli-ux/styled/table.js:76:47)
    at Object.table (/Users/xakraz/Sources/github.com/my-cli/node_modules/@oclif/core/lib/cli-ux/styled/table.js:279:39)
    at ListCommand.run (/Users/xakraz/Sources/github.com/my-cli/src/commands/list.js:56:14)
    at async ListCommand._run (/Users/xakraz/Sources/github.com/my-cli/node_modules/@oclif/command/lib/command.js:43:20)
    at async Config.runCommand (/Users/xakraz/Sources/github.com/my-cli/node_modules/@oclif/config/lib/config.js:173:24)
    at async Main.run (/Users/xakraz/Sources/github.com/my-cli/node_modules/@oclif/command/lib/main.js:27:9)
    at async Main._run (/Users/xakraz/Sources/github.com/my-cli/node_modules/@oclif/command/lib/command.js:43:20)
```

**Comment by Evanion** [comment]
@xakraz make sure you have the latest version of `@oclif/core` I noticed that the oclif generator sets `@oclif/core@^1` that resolves to `1.0.x` and `cli-ux` is available in later versions only.
